DQM/RPCMonitorClient/src/RPCEfficiencySecond.cc

```diff
@@ -200,10 +200,17 @@ void  RPCEfficiencySecond::dqmEndJob(DQMStore::IBooker & ibooker, DQMStore::IGet
 	int sector = rpcId.sector();
 	int station = rpcId.station();
 	int geolayer = rpcId.layer();
+
+	RPCBookFolderStructure *  folderStr = new RPCBookFolderStructure(); 
```

Comment:
I'll make the changes requested and push them today.

Cheers,
Anna
________________________________
From: deguio [notifications@github.com]
Sent: 02 February 2015 22:59
To: cms-sw/cmssw
Cc: Anna Cimmino
Subject: Re: [cmssw] RPC DQM Efficiency fix for re4 (#7484)


In DQM/RPCMonitorClient/src/RPCEfficiencySecond.cc<https://github.com/cms-sw/cmssw/pull/7484#discussion_r23962805>:

> @@ -200,10 +200,17 @@ void  RPCEfficiencySecond::dqmEndJob(DQMStore::IBooker & ibooker, DQMStore::IGet
>       int sector = rpcId.sector();
>       int station = rpcId.station();
>       int geolayer = rpcId.layer();
> +
> +     RPCBookFolderStructure *  folderStr = new RPCBookFolderStructure();


make sure this is deleted otherwise it leaks

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/7484/files#r23962805>.